c_src/membrane_ffmpeg_video_filter_plugin/text_overlay.c

```diff
@@ -3,14 +3,16 @@
 int get_pixel_format(char *fmt_name);
 static int init_filters(const char *filters_descr, State *state);
 void create_filter_description(char *filter_descr, int len, char *text,
-                               int fontsize, int box, char *boxcolor,
-                               int border, char *fontcolor, char *fontfile,
-                               char *horizontal_align, char *vertical_align);
+                               int fontsize, char *fontcolor, char *fontfile,
+                               int box, char *boxcolor, int borderw,
+                               char *bordercolor, char *horizontal_align,
+                               char *vertical_align);
 
 UNIFEX_TERM create(UnifexEnv *env, char *text, int width, int height,
-                   char *pixel_format_name, int fontsize, int box,
-                   char *boxcolor, int border, char *fontcolor, char *fontfile,
-                   char *horizontal_align, char *vertical_align) {
+                   char *pixel_format_name, int fontsize, char *fontcolor,
+                   char *fontfile, int box, char *boxcolor, int borderw,
+                   char *bordercolor, char *horizontal_align,
+                   char *vertical_align) {
```

Comment:
Hmm, I know it's already long, but still I'd use `border_width` as a name instead of `borderw` since it's not passed as a variable and `borderw` is only a string in filter definition.
Alternatively, you may consider using struct/map from unifex: https://hexdocs.pm/unifex/Unifex.Specs.DSL.html#type/1